sys/netpfil/pf/pf_norm.c

```diff
@@ -1478,10 +1478,10 @@ pf_normalize_tcp(struct pfi_kkif *kif, struct mbuf *m, int ipoff,
 	    (tcp_get_flags(th) & (TH_RES1|TH_RES2|TH_RES2)) != 0) {
 		u_int16_t	ov, nv;
 
-		ov = *(u_int16_t *)(&th->th_ack + 1);
+		ov = *(u_int16_t *)(__unbounded_addressof(th->th_ack) + 1);
```

Comment:
Hmm, maybe we should just not require subobject bounds on any members of `struct tcphdr`?  There's really no good reason to narrow bounds to any individual members of this structure.  I worry that other places will be doing this same trick to cope with the 12 bit flags field, so would rather fix it in the definition of `struct tcphdr` if possible.